### PR TITLE
Merge Woboq code browser page into "Getting Started" document

### PR DESCRIPTION
**Pull request by rschu1ze**
There was too little information on the page to deserve a separate document.

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: docs/ru/development/developer-instruction.md [hunk 1]
```diff
@@ -285,3 +285,11 @@ Pull request можно создать, даже если работа над з
...
 
 Вероятнее всего, часть сборок не будет успешной с первого раза. Ведь мы проверяем сборку кода и gcc и clang, а при сборке с помощью clang включаются почти все существующие в природе warnings (всегда с флагом `-Werror`). На той же странице, вы сможете найти логи сборки - вам не обязательно самому собирать ClickHouse всеми возможными способами.
+
+## Навигация по коду ClickHouse {#navigatsiia-po-kodu-clickhouse}
+
+Для навигации по коду онлайн доступен **Woboq**, он расположен [здесь](https://clickhouse.com/codebrowser/ClickHouse/src/index.html). В нём реализовано удобное перемещение между исходными файлами, семантическая подсветка, подсказки, индексация и поиск. Слепок кода обновляется ежедневно.
+
+Также вы можете просматривать исходники на [GitHub](https://github.com/ClickHouse/ClickHouse).
+
+Если вы интересуетесь, какую среду разработки выбрать для работы с ClickHouse, мы рекомендуем CLion, QT Creator, VSCode или KDevelop (с некоторыми предостережениями). Вы можете использовать свою любимую среду разработки, Vim и Emacs тоже считаются.
```
Comment (evillique) [inline docs/ru/development/developer-instruction.md:295]: Should this part with IDE suggestions be also deleted as in the English version? (There is also a whole IDE suggestions section on this page)

**Comment by rschu1ze** [comment]
The "DocsCheck" in CI failed with `[ERROR] Error: You are trying to create client-side redirections to paths that do not exist: - /en/development/browse-code`. The only reference to “browse-code” I found was in file “docs/redirects.txt” but even after deleting the corresponding line from this file the error persists in CI.

Installed the [documentation pipeline](https://github.com/ClickHouse/ClickHouse/blob/master/docs/tools/README.md) locally but the error did not reproduce, i.e. the documentation rendered as expected. I am running out of ideas 😞 .

Guess we can at least create in this PR new sections in the "developer-instruction" documents. I would then create an issue in the [documentation repo](https://github.com/ClickHouse/clickhouse-docs/issues) and ask to delete the "Woboq" pages by the doc colleagues - perhaps there is a more involved problem going on here.

**Comment by DanRoscigno** [comment]
Thanks @rschu1ze I will take it from here, when I looked at your PR over the weekend I did not see the error message locally either.  I think it was because the redirects file is only processed when `yarn build` is run (not yarn start), and I removed yarn build from the docs README as I did not expect that it would be needed outside of the build process. I will add a note about removing pages.

**Comment by rschu1ze** [comment]
Good to know, thanks. Indeed, I did not run `yarn build` locally ...

**Pull request merged**